src/parser/core/modules/Procs.tsx

```diff
@@ -331,6 +331,12 @@ export abstract class Procs extends Analyser {
 	*/
 	protected jobSpecificOnConsumeProc(_procGroup: ProcGroup, _event: Events['action']): void { /* */ }
 
+	/**
+	 * May be overriden by Subclasses. Called by OnProcGained to allow jobs to implment job-specific logic for evaluting a proc when it is gained
+	 * @param event The event to check
+	 * @returns False by default. Jobs may override to return true, allowing them to implement job-specific logic to consider an event
+	 */
+	protected jobSpecificOnProcGainedConsiderEvent(_event: Events['statusApply']): boolean { return true }
```

Comment:
The doc comment here looks to be backwards
```suggestion
	 * @returns True by default. Jobs may override to return false, allowing them to implement job-specific logic to ignore a proc gain
	 */
	protected jobSpecificOnProcGainedConsiderEvent(_event: Events['statusApply']): boolean { return true }
```